docs/mdoc/index.md

```diff
@@ -53,6 +53,8 @@ object Main extends App {
 }
 ```
 
+Try also [our giter8 template](https://github.com/finch/finch.g8) to get started: `sbt new finch/finch.g8`.
```

Comment:
that one requires some update before being mentioned in a doc

Reply:
I'll drop it, but I think it would be nice to have it here once it is updated.